src/scala/com/github/johnynek/bazel_deps/DepsModel.scala

```diff
@@ -482,6 +482,24 @@ object Language {
     def unmangle(m: MavenCoordinate) = m
   }
 
+  case object Kotlin extends Language {
```

Comment:
seems the same as java. Can we make something like:
```scala
trait JavaLike extends Language {

}
```
that everything other than the asString and share the code?